MiniSpotify/MiniSpotify/Source/APIRequestor.cs

```diff
@@ -52,6 +52,9 @@ public static APIRequestor Instance
         private string m_clientID = "93f2598a9eaf4056b34f7b5ca254ff17";
         private string m_clientSecret = "";
         private string m_clientSecretPath = "\\Assets\\Files\\ClientSecret.txt";
+        
+        private string t_albumimage = "";
```

Comment:
```suggestion
        private string m_albumimage = "";
```

Let's make this fit with the coding standards followed in the rest of the file please.